pyartcd/pyartcd/pipelines/update_golang.py

```diff
@@ -372,15 +372,12 @@ async def update_golang_streams(self, go_version, builder_nvrs):
                 output.seek(0)
                 fork_file = fork_repo.get_contents("group.yml", ref=branch_name)
                 fork_repo.update_file("group.yml", update_message, output.read(), fork_file.sha, branch=branch_name)
-            # create pr
-            try:
+		# create pr
```

Comment:
I think the code block originally under `try:` should be in the same indentation level as `try:` was.